pom.xml

```diff
@@ -5,10 +5,10 @@
 
 	<groupId>life.qbic</groupId>
 	<artifactId>omero-client-lib</artifactId>
-	<version>1.2.1</version>
+	<version>1.2.2</version>
```

Comment:
Since you change existing behaviour with a new feature i'd not do an increment release version but a minor one(1.3.0 instead of 1.2.2.)